rootfs/start.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -z ${EXECUTE_DEV_INSTALL+x} ]; then
+    composer install --no-interaction
+    composer clear-cache
+fi
+if [ ! -z ${EXECUTE_LARAVEL_OPTIMIZATION+x} ]; then
```

Comment:
I added a new `ENV` variable to optimize Laravel code when we deploy in a production environment. For dev environment we can omit it